### PR TITLE
Update `HelpWindow` UI

### LINKED ISSUE #176: Update `help` window content in GUI

### PR DESCRIPTION
**Pull request by LeeEnEn**
Closes #176

### CODE REVIEW

**Review by domlimm — APPROVED** [review]
Everything LGTM! Thank you so much for making the change! Such a pro @LeeEnEn! Darn good job! 😄 💯

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -17,25 +17,37 @@
...
-            + "- exit\n\n"
-            + "The full guide is available on our User Guide";
-
+    private static final String HELP_MESSAGE = "Available functions on this version: \n\n";
+    private static final String ADD_FIELD = "n/NAME id/STUDENT_ID c/COURSE avail/AVAILABILITY";
+    private static final String DELETE_FIELD = "INDEX";
+    private static final String EDIT_FIELD = "INDEX [PREFIX]/[NEW VALUE]";
+    private static final String FIND_FIELD = "k/KEYWORD k/[MORE_KEYWORDS] f/[ATTRIBUTE_FIELD]";
+    private static final String FOCUS_FIELD = "INDEX";
+    private static final String SORT_FIELD = "s/ATTRIBUTE";
```
Comment (charmainehly) [inline src/main/java/seedu/address/ui/HelpWindow.java:26]: Sorry @LeeEnEn not sure if i missed it out, will we be including commands like `view` and `schedule`?

File: src/main/resources/view/HelpWindow.css [hunk 2]
```diff
@@ -17,3 +17,28 @@
...
+    -fx-background-color: #3F7BCD;
+    -fx-border-radius: 2;
+    -fx-background-radius: 3;
+    -fx-font-size: 14;
+    -fx-padding: 3 5 3 5;
+}
+
+.styleHBox {
+    -fx-padding: 8 0 0 10;
+}
```
Comment (charmainehly) [inline src/main/resources/view/HelpWindow.css:44]: Thanks for updating the UI !!! :)

**Review by charmainehly — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (LeeEnEn) [reply src/main/java/seedu/address/ui/HelpWindow.java:26]: Yeps, I can add those in. 😄

Comment (charmainehly) [reply src/main/java/seedu/address/ui/HelpWindow.java:26]: Thank you!!!! 😄

**Review by charmainehly — APPROVED** [review]
LGTM thank you !!!

**Pull request merged**